gen/flights/ai_flight_planner_db.py

```diff
@@ -209,6 +210,7 @@
     WingLoong_I,
     MQ_9_Reaper,
     RQ_1A_Predator,
+    S_3B,
```

Comment:
This list is in priority order. I don't think the S-3 is the worst choice for CAS. Not sure what loadout capabilities it has, but it's probably a better choice than most of the helicopters?

Reply:
It is indeed, didn't know about the priority order, I'll check for a better spot